docs/modules/ROOT/partials/getting-started/ref-registry-security-configuration.adoc

```diff
@@ -100,9 +154,71 @@ When {registry} authentication is enabled, you must assign {registry} users to a
 |Yes
 |No
 |No
-|Access to read and search operations only. This role cannot configure any rules. 
+|Access to read and search operations only. This role cannot configure any rules.
 |===
 
+[discrete]
+=== Managing roles directly in {registry}
+
+To enable using roles managed internally by {registry}, set the following environment variables:
+
+.Configuration for {registry} role based authorization using internal role mappings
+[.table-expandable,width="100%",cols="6,6,2,3",options="header"]
+|===
+|Environment variable
+|Description
+|Type
+|Default
+|`ROLE_BASED_AUTHZ_SOURCE`
+| When set to `application`, user roles will be managed internally.
+|String
+|`token`
+|===
+
+When using internally managed role mappings, users can be assigned a role using the `/admin/roleMappings`
+endpoint in the {registry} REST API.  See the REST API documentation for details on how to use that API.
+Users can be granted exactly one role:  `ADMIN`, `DEVELOPER`, or `READ_ONLY`.  Only users with Admin
+privileges can grant access to other users.  Because there are no default Admin users in {registry}, it
+is usually helpful to configure another way for users to be identified as admins.  This admin-override
+configuration can be controlled with the following environment variables:
+
+.Configuration for {registry} admin-override functionality
+[.table-expandable,width="100%",cols="6,6,2,3",options="header"]
+|===
+|Environment variable
+|Description
+|Type
+|Default
+|`REGISTRY_AUTH_ADMIN_OVERRIDE_ENABLED`
+| Enables the admin-override feature.
+|String
+|`false`
+|`REGISTRY_AUTH_ADMIN_OVERRIDE_FROM`
+|Where to look for admin-override information.  Only `token` is currently supported.
+|String
+|`token`
+|`REGISTRY_AUTH_ADMIN_OVERRIDE_TYPE`
+|The type of information used to determine if a user is an Admin.  Values depend on the FROM above.  E.g. `role` or `claim` when FROM is `token`.
+|String
+|`role`
+|`REGISTRY_AUTH_ADMIN_OVERRIDE_ROLE`
+|The name of the role that indicates a user is an Admin.
+|String
+|`sr-admin`
+|`REGISTRY_AUTH_ADMIN_OVERRIDE_CLAIM`
+|The name of a JWT token claim to use for determining admin-override.
+|String
+|`org-admin`
+|`REGISTRY_AUTH_ADMIN_OVERRIDE_CLAIM-VALUE`
+|The value that the JWT token claim indicated by CLAIM must be for the user to be granted admin-override.
+|String
+|`true`
+|===
+
+For example, the admin-override feature would allow you to assign the `sr-admin` role to a single user
+in {keycloak} which would grant that user the Admin role.  That user could then use the `/admin/roleMappings`
+REST API (or associated UI) to grant roles to additional users (including additional admins).
```

Comment:
Suggest the following:

For example, you can use the admin-override feature to assign the `sr-admin` role to a single user
in {keycloak}, which grants that user the admin role.  That user can then use the `/admin/roleMappings`
REST API (or associated UI) to grant roles to additional users (including additional admins).